### ISSUE TITLE
Replace the image on the join page, possibly?

### ISSUE THREAD

**Issue by isdampe** [DeakinSEC/legacy-website#8]
Currently, the image on the join page is as per attached.

![screen shot 2018-02-26 at 9 34 15 pm](https://user-images.githubusercontent.com/7271686/36665887-e78f7310-1b3c-11e8-8b8c-4e932cda5a8d.png)

We're currently hot-linking to a third party directly. 

```html
    <section class='content container'>
      <h1>Come and Join DSEC!</h1>
      <img src="http://www.epilepsyconsortiumscotland.co.uk/wp-content/uploads/2016/03/join-us.jpg">
    </section>
```

I'm also unsure about the license requirements for using this image.

**Comment by PhilipCastiglione** [comment]
Yep this is not a good idea long term. Ideally we can use a photo we take from an event of our own!

**Comment by isdampe** [comment]
@mjocen may have some photos from OWeek that can be used short-term

**Comment by isdampe** [comment]
Fixed by ff18cd01b3a0c53b5078810104393144dc72fa71 in develop
